pkg/wallet/wallet.go

```diff
@@ -48,6 +50,46 @@ type Wallet struct {
 	KeyPair  KeyPair
 }
 
+type AccountSerialized struct {
+	Version      uint8
```

Comment:
Shouldn't Version be in string format to follow the semantic versioning!? (1.2.3 - 2.2.2, etc) 🤔 :

MAJOR version when you make incompatible API changes
MINOR version when you add functionality in a backwards compatible manner
PATCH version when you make backwards compatible bug fixes

Reply:
Furthermore, it's following the Wallet file format standard defined [here](https://github.com/massalabs/massa-standards/blob/main/wallet/file-format.md)

Reply:
No worries, folks! It's up to you!

It was just a suggestion! Closing...

Reply:
Ops... was already closed! 😞